lib/DDG/Spice/Thumbtack.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::Thumbtack;
+# ABSTRACT: Returns service providers nearby
+
+use DDG::Spice;
+
+name "Thumbtack Search";
+source "Thumbtack";
+icon_url "https://www.thumbtack.com/favicon.ico";
+description "Shows a list of service providers";
+primary_example_queries "plumbers near me", "electricians close by";
+category "location_aware";
+topics "everyday";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Thumbtack.pm";
+attribution github => ["http://github.com/whalenrp", "whalenrp"];
+
+# Require the "thumbtack " modifier explicitly for now.
+my @keywords = share('top_keywords.txt')->slurp;
+for (my $i=0; $i < scalar @keywords; $i++) {
+    $keywords[$i] = "thumbtack " . $keywords[$i]
```

Comment:
@whalenrp this looks good. We could make it more flexible by making `thumbtack` a startend trigger and checking if the remainder is one of the words in our list. We could also strip out weasel words like "on" "for" etc to handle queries like "electricians on thumbtack"

This is perfectly fine for now though!